js/scripts.js

```diff
@@ -0,0 +1,14 @@
+var a = '2';
+    b = '4';
+    value = (a * a) + (2 * a * b) - (b * b);
+
+console.log('Wynik: '+value+' ');
+
+
+if (value > 0) {
+    console.log('wynik dodatni');
+}   else if (value < 0) {
+    console.log('wynik ujemny');
+}   else if (value = 0) {
```

Comment:
po co `else if` zamiast `else` ? Tam już nie ma co sprawdzać przecież. Swoją drogą sprawdzanie robimy poprzez `===`. 1 x `=` to przypisanie wartości, a nie jej sprawdzenie.